controllers/kubeletconfig_controller.go

```diff
@@ -288,6 +285,12 @@ func (r *KubeletConfigReconciler) makeKCHandlerForPlatform(ctx context.Context,
 			ownerObject: cmKc,
 			mcoKc:       mcoKc,
 			poolName:    nodePoolName,
+			// the owner should be the KubeletConfig object and not the NUMAResourcesOperator CR
+			// this means that when KubeletConfig will get deleted, the ConfigMap gets deleted as well
+			// TODO on HyperShift there's a cross-namespaced owner references that need to be fixed.
```

Comment:
We shall remove this line because we added https://github.com/openshift-kni/numaresources-operator/pull/1088